hr_expense_petty_cash/readme/USAGE.rst

```diff
@@ -0,0 +1,25 @@
+**Create a Petty Cash Holder**
+
+#. Go to Invoicing > Configuration > Petty Cash
+#. Create a new petty cash holder
+#. Select Petty Cash Account
+
+**Add Balance for Petty Cash Holder**
+
+#. Go to Invoicing > Vendor > Bill
+#. Create a new vendor bill
+#. Select Vendor (Petty Cash Holder)
+#. Check Petty Cash will auto line with Unit Price = Max Limit - Balance
+
+**Create an Expense paid by Petty Cash**
+
+#. Go to Expenses > My Expenses
+#. Create a new Expense
+#. Select Paid by = Petty Cash and Select Petty Cash Holder
+
+**Create an Expense Report paid by Petty Cash**
```

Comment:
Do you repeat content in `Create an Expense paid by Petty Cash` ?

Reply:
@ps-tubtim would you take a look at this?

Reply:
@Saran440 @kittiu I edited it all. Please review again. Thank you.